### PR TITLE
Fix AFL showmap call.

### LINKED ISSUE #2703: AFL++ fuzzers not uploading corpus
From Jonathan Metzman:
Nothing is getting added to corpus because in the merge implementation ClusterFuzz thinks every testcase covers the same code

### PR DESCRIPTION
**Pull request by oliverchang**
AFL driver arguments changed. 

Fixes #2703.

Also rebuild test fuzzers using latest OSS-Fuzz toolchain.

### CODE REVIEW

**Comment by oliverchang** [comment]
@jonathanmetzman not sure if I messed this up, but compiling the `always_crash_fuzzer` and `easy_crash_fuzzer` like so:

```
/src/aflplusplus/afl-clang-fast++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope  -stdlib=libc++ easy_crash_fuzzer.cc /src/aflplusplus/libAFLDriver.a -o easy_crash_fuzzer
```

I can't get the resulting binary to crash at all with the expected input. libFuzzer works fine. 

Any ideas?

**Comment by oliverchang** [comment]
Looks like it was some kind of weird optimization that completely removed the vulnerable code call. Adding some printfs seems to fix this.

File: src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ def merge_corpus(self):
     self._executable_path = self.afl_showmap_path
     # Hack around minijail.
     showmap_args = self._fuzz_args
-    showmap_args[-1] = '1'
+    showmap_args[-1] = '-'
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py:1033]: This fix looks correct to me.

File: src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_launcher_integration_test.py [hunk 2]
```diff
@@ -207,9 +207,6 @@ def _test_libfuzzerize_corpus(self, mock_get_timeout):
             # 'idCOLON000000,larger_same_cov_idCOLON000001,non_initial_cov'.
             # This filename is the hash of idCOLON000001,non_initial_cov.
             '3dce8306f3c1810d5d81ed5ebb0ccea947277a61',
-            # This file reaches unique paths and should therefore be merged.
-            # This filename is the hash of idCOLON000002,unique_cov.
-            '4c1fe29b1a967d34a6fcc078b8fb653dd807dee7',
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_launcher_integration_test.py:212]: I don't have the time at this late hour for me to figure out if this test is still working properly. I can try tomorrow

[the remark this answers is not in the document]
Comment (oliverchang) [reply src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_launcher_integration_test.py:212]: SG. we can investigate this later :)

**Pull request merged**